todo_app/data/trello_items.py

```diff
@@ -0,0 +1,102 @@
+import requests
+import os
+from todo_app.data.item import Item
+
+key = os.getenv('TRELLO_API_KEY')
+token = os.getenv('TRELLO_API_TOKEN')
+board_id = os.getenv('TRELLO_BOARD_ID')
+
+auth_params = {
+    'key': key,
+    'token': token
+}
+
+def get_list_id(list_name):
+    url = (f'https://api.trello.com/1/boards/{board_id}/lists')
+
+    headers = {
+        'Accept': 'application/json'
+    }
+
+    params = {
+        **auth_params,
+        'cards': 'open',
+        'fields': 'name'
+    }
+
+    response = requests.request(
+        'GET',
+        url,
+        headers=headers,
+        params=params
+    )
+
+    for trello_list in response.json():
+        if trello_list['name'] == list_name:
+            return trello_list['id']
```

Comment:
Any indented return makes me think it would be nice to have some sort of fall back behaviour if no list matches - even if it's just throw an error you ignore elsewhere!